asv_bench/benchmarks/utils/common.py

```diff
@@ -455,23 +455,30 @@ def trigger_import(*dfs):
     ----------
     *dfs : iterable
         DataFrames to trigger import.
+
+    Returns
+    -------
+    tuple of DataFrames
+        Returns exactly the same objects that were passed into the function.
     """
-    assert ASV_USE_STORAGE_FORMAT == "omnisci"
+    if ASV_USE_STORAGE_FORMAT != "omnisci" or ASV_USE_IMPL != "modin":
+        return
 
     from modin.experimental.core.execution.native.implementations.omnisci_on_native.omnisci_worker import (
         OmnisciServer,
     )
 
     for df in dfs:
-        if ASV_USE_IMPL == "modin":
-            df.shape  # to trigger real execution
-            df._query_compiler._modin_frame._partitions[0][
-                0
-            ].frame_id = OmnisciServer().put_arrow_to_omnisci(
-                df._query_compiler._modin_frame._partitions[0][0].get()
-            )  # to trigger real execution
-        elif ASV_USE_IMPL == "pandas":
-            pass
+        if not isinstance(df, (pd.DataFrame, pd.Series)):
+            continue
+        df.shape  # to trigger real execution
+        df._query_compiler._modin_frame._partitions[0][
+            0
+        ].frame_id = OmnisciServer().put_arrow_to_omnisci(
+            df._query_compiler._modin_frame._partitions[0][0].get()
+        )  # to trigger real execution
+
+    return dfs
```

Comment:
adding return here to ease variables initialization by already imported frames (see [here](https://github.com/dchigarev/modin/blob/817ca9b1d5de9a538346880ce9c26d68457daa47/asv_bench/benchmarks/benchmarks.py#L518))